components/vocabularies/omim/api/src/main/java/org/phenotips/vocabulary/internal/solr/MendelianInheritanceInMan.java

```diff
@@ -59,8 +61,21 @@
 @Singleton
 public class MendelianInheritanceInMan extends AbstractSolrVocabulary
 {
+    private static final String DISEASE = "disease";
+
+    private static final String GENE = "gene";
+
+    /** The list of supported categories for this vocabulary. */
+    private static final Collection<String> SUPPORTED_CATEGORIES = Arrays.asList(DISEASE, GENE);
+
     /** The standard name of this vocabulary, used as a term prefix. */
-    public static final String STANDARD_NAME = "MIM";
+    private static final String STANDARD_NAME = "MIM";
+
+    /** The default filter for disease OMIM vocabulary searches. */
+    private static final String DEFAULT_DISEASE_FILTER = "-(nameSort:\\** nameSort:\\+* nameSort:\\^*)";
+
+    /** The default filter for GENE OMIM vocabulary searches. */
+    private static final String DEFAULT_GENE_FILTER = "-(nameSort:\\%* nameSort:\\#* nameSort:\\^*)";
```

Comment:
I think it would be safer to say `nameSort:\\** nameSort:\\+*`. The negative filter for diseases was used to also allow entries without a starting symbol, but for genes only these two known symbols can be used.